### PR TITLE
VC+BN: Validator voluntary exits through the Keymanager API

### PR DESCRIPTION
**Pull request by cheatfate**

### CODE REVIEW

File: beacon_chain/rpc/rest_key_management_api.nim [hunk 1]
```diff
@@ -522,3 +525,53 @@ proc installKeymanagerHandlers*(router: var RestRouter, host: KeymanagerHost) =
       response.data.add handleRemoveValidatorReq(host, key)
 
     return RestApiResponse.jsonResponsePlain(response)
+
```
Comment (arnetheduck) [inline beacon_chain/rpc/rest_key_management_api.nim:528]: link to upstream API?

**Review by arnetheduck — APPROVED** [review]

File: beacon_chain/rpc/rest_key_management_api.nim (hunk 1 above)
Comment (cheatfate) [reply beacon_chain/rpc/rest_key_management_api.nim:528]: Specification changes got merged into master with https://github.com/ethereum/keymanager-APIs/commit/ee334a457a95b5ee0230c2f4603efc45e9830a1f but there no link yet.

Comment (etan-status) [reply beacon_chain/rpc/rest_key_management_api.nim:528]: https://ethereum.github.io/keymanager-APIs/?urls.primaryName=dev#/Voluntary%20Exit/signVoluntaryExit

Comment (tersec) [reply beacon_chain/rpc/rest_key_management_api.nim:528]: > Specification changes got merged into master with [ethereum/keymanager-APIs@ee334a4](https://github.com/ethereum/keymanager-APIs/commit/ee334a457a95b5ee0230c2f4603efc45e9830a1f) but there no link yet.

Also, https://github.com/ethereum/keymanager-APIs/pull/58 or that is a link, and better than nothing.

**Pull request merged**